src/app_service/service/contacts/dto/contacts.nim

```diff
@@ -77,12 +77,12 @@ proc toImages(jsonObj: JsonNode): Images =
 
 proc toContactRequestState*(value: int): ContactRequestState =
   result = ContactRequestState.None
-  if value >= ord(low(ContactRequestState)) or value <= ord(high(ContactRequestState)):
+  if value >= ord(low(ContactRequestState)) and value <= ord(high(ContactRequestState)):
       result = ContactRequestState(value)
 
 proc toTrustStatus*(value: int): TrustStatus =
   result = TrustStatus.Unknown
-  if value >= ord(low(TrustStatus)) or value <= ord(high(TrustStatus)):
+  if value >= ord(low(TrustStatus)) and value <= ord(high(TrustStatus)):
```

Comment:
Nice find. This could have made the app crash :grimacing:

Reply:
Right, I forgot to mention that! I assume this could very well crash if it was given an out of range value